src/Submit.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Button from './Button';
+import FormGroup from './FormGroup';
+import InputBase from './InputBase';
+
+class Submit extends InputBase {
+  renderFormGroup(children) {
+    let {bsStyle, ...other} = this.props; /* eslint no-unused-vars: 0 */
```

Comment:
@AlexKVal Did you create a bug with eslint about this use case? If so we should reference the bug in a comment here so we know to remove this eslint exception at some point.